pages/about.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <!-- Adaugam metatag-urile pentru Google -->
-    <title>Clonă IT School</title>
+    <title>Despre noi</title>
```

Comment:
You've identified right the title attribute. However, The value is not the desired one. You should use the value from the h1 tag (the big title on the page).

Reply:
Done